doc/api/vm.md

```diff
@@ -107,6 +110,16 @@ changes:
     experimental modules API. We do not recommend using it in a production
     environment. For detailed information, see
     [Support of dynamic `import()` in compilation APIs][].
+  * `transform` {Object} An object containing options for type stripping.
+    Only available when the `--experimental-strip-types` flag is enabled.
+    * `mode` {string} The mode of type stripping. Possible values are:
+      * `strip-only`: Strip all types, without transforming TypeScript only features,
+        for more information read [type-stripping][].
+      * `transform`: Transforms TypeScript features, for info read [transform TypeScript features][].
+    * `sourceMap` {boolean} Generate source maps for the transformed code, if `mode` is `transform`,
+      otherwise it is not necessary.
+      The filename for the source map is the same as the filename of the script.
+      **Default:** `false`.
```

Comment:
```suggestion
	* `transform` {Object} An object that holds options for type stripping. This is accessible only when the `--experimental-strip-types` flag is enabled.
	  * `mode` {string} Specifies the type stripping mode. The possible options are:
	    * `strip-only`: Removes all types without altering TypeScript-specific features. For more details, refer to [type-stripping][].
	    * `transform`: Modifies TypeScript features. For additional information, see [transform TypeScript features][].
	  * `sourceMap` {boolean} If `mode` is set to `transform`, generates source maps for the transformed code; otherwise, it is unnecessary. The source map will have the same filename as the script.  
	  **Default:** `false`.
```

Reply:
What is the change?

Reply:
It's just worded a bit different. IMO the change makes it more understandable, but feel free to ignore:

E.G.:
``An object containing options for type stripping.
    Only available when the `--experimental-strip-types` flag is enabled.`` becomes ``An object that holds options for type stripping. This is accessible only when the `--experimental-strip-types` flag is enabled.``

Reply:
This is a draft PR why are you nitpicking on wording?

Reply:
Sorry, just my two-cents, I'll leave it be.